### ISSUE TITLE
[COGNITION]: EYB - CONSIDER updating link text "this page" to more descriptive labels

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#9243]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️  Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Some of our modals have links that read "this page" after a descriptive sentence or two. It might be worth considering updating these links to be more descriptive. Screenshot attached below.

/cc @peggygannon for awareness.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Link text is clear and offers enough information to users without needing to rely on the text around them

## Supporting Artifacts: 
1. Content for all links included here: 
[8b8ae70e-d776-4303-b16e-fce0a8f2e72d.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/4713270/8b8ae70e-d776-4303-b16e-fce0a8f2e72d.docx)

## WCAG or Vendor Guidance (optional)

* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-18 at 9 24 20 PM](https://user-images.githubusercontent.com/934879/82278026-dfd50580-994e-11ea-8fbb-0e785dbe2598.png)

---
![Screen Shot 2020-05-27 at 12 45 59 PM](https://user-images.githubusercontent.com/934879/83055451-74a7c500-a019-11ea-97b0-63fda590d607.png)

---
![Screen Shot 2020-05-27 at 12 46 13 PM](https://user-images.githubusercontent.com/934879/83055462-7a050f80-a019-11ea-9d4f-069857136fcc.png)

---
![Screen Shot 2020-05-27 at 12 57 31 PM](https://user-images.githubusercontent.com/934879/83055635-b6387000-a019-11ea-804f-639bd03851df.png)

**Comment by desireeturner** [comment]
@amyknox @ccruzgra @JenLJ This is a "consider" (re: priority) but could use your input regarding the language when you all have a chance to review. Thanks!

**Comment by 1Copenut** [comment]
@amyknox I relabeled this one `BAH-EYB` and removed `bah-section103` because I got the two conflated while working on the 103 staging review last night.

**Comment by dneel-bah** [comment]
That link actually should be removed entirely.  We are currently awaiting feedback from EDU on a total modal content update.  Once approved that link will be removed with the content update.

**Comment by amyknox** [comment]
@1Copenut 

Looks like this issue appears on 
- Which GI Bill benefit do you want to use? (Learn more) - on landing page & search results page
and
- Cumulative Post-9/11 service (Learn more) - on landing page & search results page, when Post-9/11 GI Bill is selected in the Which GI Bill benefit is selected

Are there other places you found that issue?

**Comment by amyknox** [comment]
"this page" appears as a link in the modal / helper text for these controls. 
* Which GI Bill benefit do you want to use? (all pages)
* Profile Page / About your school (EYB accordion) / Enrolled
* Profile Page / Scholarships & other funding (EYB accordion) / Eligible for kicker bonus

Cumulative Post-9/11 modal was resolved under a separate issue.

**Comment by 1Copenut** [comment]
@amyknox I used your comment here and added screen shots to the ticket description. Thank you for providing the bulleted list.

**Comment by amyknox** [comment]
[https://app.zenhub.com/files/133843125/8b8ae70e-d776-4303-b16e-fce0a8f2e72d/download](https://app.zenhub.com/files/133843125/8b8ae70e-d776-4303-b16e-fce0a8f2e72d/download)

**Comment by amyknox** [comment]
@ksantiagomorgan6 - the Post-9/11 selection modal is being handled in 9522

Check it out here: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/9522

**Comment by ksantiagomorgan6** [comment]
pull request: https://github.com/department-of-veterans-affairs/vets-website/pull/12896

**Comment by delli-gatti-michael** [comment]
Local testing complete for this one.
"Cumulative Post-9/11 service" is covered by #9522 but fixed on the same branch

![Screen Shot 2020-06-03 at 4.31.29 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/40190d96-9318-4f6d-ba6b-5d6ba9aea9fb)

---

![Screen Shot 2020-06-03 at 4.35.52 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/9a9b134e-dbeb-4684-a271-5c1fd3b72317)

---

![Screen Shot 2020-06-03 at 4.41.38 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/56010797-428b-4873-8822-0641b205152d)

---

![Screen Shot 2020-06-03 at 4.42.08 PM.png](https://images.zenhubusercontent.com/5c07f000391a6f6357adc430/3453bd99-5e6e-4ccd-ab15-78e714c6dce8)

**Comment by delli-gatti-michael** [comment]
@department-of-veterans-affairs/uxbah This is in staging and ready for review

**Comment by ccruzgra** [comment]
@delli-gatti-michael @JenLJ and I took a look, and this checks out!

**Comment by delli-gatti-michael** [comment]
@1Copenut This one is ready for your review on Staging

**Comment by 1Copenut** [comment]
@delli-gatti-michael LGTM. Moving this one to close.
